### PR TITLE
Resolve XHTML5 compatibility issues highlighted by https://validator.…

### PR DESCRIPTION
**Pull request by minfrin**
…w3.org:

- Remove unnecessary type from script tags.
- Remove scrolling/frameborder from iframe.
- Use HTML5 compliant html tag.
- Assign appropriate header/nav/main/footer tags in the site.vm.

https://jira.apache.org/jira/browse/MSKINS-153

### CODE REVIEW

**Comment by michael-o** [comment]
Will look into it.

File: src/main/resources/META-INF/maven/site.vm [hunk 1]
```diff
@@ -28,10 +28,10 @@
  | Rendered using Apache Maven Fluido Skin @project.version@
 -->
 ##
-<html xmlns="http://www.w3.org/1999/xhtml"#if ( $locale ) xml:lang="$locale.language" lang="$locale.language"#end>
```
Comment (michael-o) [inline src/main/resources/META-INF/maven/site.vm]: I am not certain about `xml:id` but dropping the namespace seems wrong. See [here](https://stackoverflow.com/a/34407786/696632) and [here](https://github.com/whatwg/html/blob/master/FAQ.md#what-is-the-namespace-declaration).

File: src/main/resources/META-INF/maven/site.vm [hunk 2]
```diff
@@ -137,6 +137,11 @@
             </ul>
 #*      *##end
 ##
+#*      *##if (!$sideBarEnabled)
+          </nav>
+          <div class="nav-collapse">
```
Comment (michael-o) [inline src/main/resources/META-INF/maven/site.vm]: I am confused about this. Why is this ecessary now and why not as `nav`?

File: src/main/resources/META-INF/maven/site.vm [hunk 3]
```diff
@@ -161,82 +166,86 @@
...
       <div class="row-fluid">
-        <div id="leftColumn" class="$leftColumnClass">
-          <div class="well sidebar-nav">
+        <header id="leftColumn" class="$leftColumnClass">
+          <nav class="well sidebar-nav">
 #*      *##publishDate( "navigation-top" $decoration.publishDate $decoration.version )
 #*      *##mainMenu( $decoration.body.menus )
 #*      *##publishDate( "navigation-bottom" $decoration.publishDate $decoration.version )
+          </nav>
+          <div class="well sidebar-nav">
```
Comment (michael-o) [inline src/main/resources/META-INF/maven/site.vm]: Why is this not a `nav` too?

File: src/main/resources/META-INF/maven/site.vm (hunk 1 above)
Comment (minfrin) [reply src/main/resources/META-INF/maven/site.vm]: This one was overzealous. I've put it back.

File: src/main/resources/META-INF/maven/site.vm (hunk 3 above)
Comment (minfrin) [reply src/main/resources/META-INF/maven/site.vm]: I understand <nav> is supposed to surround navigation elements, but specifically exclude non -navigation links. The links excluded are things like the google search, facebook like button, etc, none of which are related to navigation.

File: src/main/resources/META-INF/maven/site.vm (hunk 2 above)
Comment (minfrin) [reply src/main/resources/META-INF/maven/site.vm]: As below...

File: src/main/resources/META-INF/maven/site.vm [hunk 4]
```diff
@@ -161,82 +166,86 @@
             </ul>
 #*      *##end
 ##
-#*      *##if ($topBarEnabled)
-            </div>
+#*      *##if (!$sideBarEnabled)
```
Comment (michael-o) [inline src/main/resources/META-INF/maven/site.vm]: Stupid question: Was has the `$topBarEnabled` been exchanged for `!$sideBarEnabled`?

Comment (minfrin) [reply src/main/resources/META-INF/maven/site.vm]: That was a bug I found - the open tag appears if !$sideBarEnabled, but the close tag didn't have the same conditional logic.

There is nothing stopping you having both true, so I polished that out.

**Comment by michael-o** [comment]
I see this locally:

```
[INFO] -------------------------------------------------                         
[ERROR] The following builds failed:                                             
[ERROR] *  mskins-75\pom.xml                                                     
[ERROR] *  mskins-76\pom.xml                                                     
[ERROR] *  mskins-76_topbar\pom.xml
```

Please have a look and rebase your branch on top of the one I have pushed.

**Comment by minfrin** [comment]
Integration tests are fixed.

**Pull request closed without merging**